server/helpers/common_registry_logs.go

```diff
@@ -70,12 +72,14 @@ func HandleError(h v1beta1.Host, en entity.Entity, err error, isModelError bool,
 		if err != nil {
 			handleModelOrRegistrantError(h, entity.Model.Name, err, isModelError, isRegistrantError)
 
+			LogHandler.RegisterAttempts[meshmodel.HostnameToPascalCase(h.Hostname)].mu.Lock()
```

Comment:
Looks good however the logic for v1beta1 and v1alpha two looks mostly similar, you can extract his logic to make sure we are not repeating, something for next PR

Reply:
okay